packages/tailwindcss-transformer/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@clerk/tailwindcss-transformer",
+  "version": "1.0.0",
+  "private": true,
+  "license": "MIT",
+  "author": {
+    "name": "Clerk, Inc.",
+    "email": "support@clerk.com",
+    "url": "git+https://github.com/clerk/javascript.git"
+  },
+  "scripts": {
+    "test": "vitest"
+  },
+  "dependencies": {
+    "@babel/parser": "^7.24.5",
+    "class-variance-authority": "^0.7.0",
+    "fast-glob": "^3.3.2",
```

Comment:
i dont think we use this one anymore

Reply:
👍🏼 https://github.com/clerk/javascript/pull/3397/commits/e454ff92945b4d512502bd8a7f975d52a2f04ae5